quinn-proto/src/crypto/rustls.rs

```diff
@@ -56,8 +56,8 @@ impl crypto::Session for TlsSession {
 
     fn initial_keys(dst_cid: &ConnectionId, side: Side) -> Keys<Self> {
         const INITIAL_SALT: [u8; 20] = [
-            0xaf, 0xbf, 0xec, 0x28, 0x99, 0x93, 0xd2, 0x4c, 0x9e, 0x97, 0x86, 0xf1, 0x9c, 0x61,
-            0x11, 0xe0, 0x43, 0x90, 0xa8, 0x99,
+            0x38, 0x76, 0x2c, 0xf7, 0xf5, 0x59, 0x34, 0xb3, 0x4d, 0x17, 0x9a, 0xe6, 0xa4, 0xc8,
```

Comment:
Can this support both versions for a while? E.g. msquic and picoquic as a server currently support both accepting v1 and draft connections. As a client they allow to specify the version number to send. 

I think there are also more salts than just this  one.

Reply:
Maybe, but supporting multiple versions in parallel like that is probably a separate project/PR.

Reply:
Maybe - but unless this is tackled this will immediately break the project for anyone who needs to interact with a draft version - which are currently e.g. all browsers.

I don't think "quinn was usable, but will stop working for the next couple of month until others have caught up" is a great story.